webapps/frontend/app/viewmodels/wizards/addvpool/index.js

```diff
@@ -61,8 +61,8 @@ define([
         data.dtlEnabled(true);
         data.dtlMode({name: 'a_sync', disabled: false});
         data.dtlTransportMode({name: 'tcp'});
-        data.fragmentCacheOnRead(true);
-        data.fragmentCacheOnWrite(false);
```

Comment:
No, the ```index.js```'s settings are used anyway, they immediately overwrite ```data.js```'s settings. So in theory, most of ```data.js```'s default could be removed at all.